core/dbt/task/run.py

```diff
@@ -481,8 +489,8 @@ def _execute_microbatch_materialization(
             start = microbatch_builder.build_start_time(end)
             batches = microbatch_builder.build_batches(start, end)
         else:
```

Comment:
Can we add a comment here mentioning this only happens during retry?

Reply:
Can do!